services/datamanager/builtin/sync_test.go

```diff
@@ -190,6 +192,10 @@ func TestDataCaptureUpload(t *testing.T) {
 			dataType: v1.DataType_DATA_TYPE_BINARY_SENSOR,
 			numFails: 2,
 		},
+		{
+			name:      "If syncing more often than capturing, empty sensor reading files should be deleted locally and not uploaded.",
```

Comment:
Sort of echoing other comment: I think the more general rule here is just about a file having no sensor data vs this relationship between capture and sync rate, so something like "Files with no sensor data should not be synced" would be a bit more informative

Reply:
+1 updated